### PR TITLE
#18/UI/NavElement

### PR DESCRIPTION
**Pull request by KatarzynaChmielecka**
image

### CODE REVIEW

File: src/Organisms/NavElement/NavElementStyled.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  }
+
+  .back {
+    font-family: Mulish;
+    font-size: 14px;
+    color: ${COLORS.grey200};
+  }
+`;
+
+export const NavElementContainer = styled.div`
```
Comment (Smyku6) [inline src/Organisms/NavElement/NavElementStyled.js:24]: change name to the same as filename - NavElementStyled

File: src/Organisms/NavElement/NavElementStyled.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  .navText {
+    color: ${COLORS.grey800};
+    width: 33%;
+    font-size: 19px;
+    font-family: Mulish;
+  }
+`;
+
+export const Logo = styled.div`
```
Comment (Smyku6) [inline src/Organisms/NavElement/NavElementStyled.js:42]: move this component (Logo) to Atoms

File: src/Organisms/NavElement/NavElementStyled.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import { COLORS } from '../../Styles/varsStyledComponents';
+
+export const LinkWrapper = styled.div`
```
Comment (Smyku6) [inline src/Organisms/NavElement/NavElementStyled.js:4]: move this component (LinkWrapper) to Atoms

File: src/Organisms/NavElement/NavElementStyled.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+export const NavElementContainer = styled.div`
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 60px;
+
```
Comment (Smyku6) [inline src/Organisms/NavElement/NavElementStyled.js:33]: add 
border: solid 1px ${COLORS.grey400};

File: src/Organisms/NavElement/NavElementStyled.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  cursor: pointer;
+
+  .arrowLeft {
+    color: ${COLORS.grey200};
+    font-size: 30px;
+  }
+
+  .back {
+    font-family: Mulish;
+    font-size: 14px;
```
Comment (Smyku6) [inline src/Organisms/NavElement/NavElementStyled.js:19]: remember- fight with REM (relative to font-size of the root element)

File: src/Organisms/NavElement/NavElementStyled.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    color: ${COLORS.grey200};
+  }
+`;
+
+export const NavElementContainer = styled.div`
+  margin: 0;
```
Comment (Smyku6) [inline src/Organisms/NavElement/NavElementStyled.js:25]: think about left and right margin of your container (I propose 1REM on the left and 1REM on the right)

**Review by n0macx — COMMENTED** [review]
Maybe You should add NavElement with proper text to every view not only home page ?

**Review by Smyku6 — CHANGES_REQUESTED** [review]
in comments

**Comment by KatarzynaChmielecka** [comment]
> Maybe there a way for the text od nav to change after changing views ?

What do you mean?

**Review by Smyku6 — DISMISSED** [review]

**Pull request closed without merging**